utils/sql/git/required/2022_04_30_hp_regen_per_second.sql

```diff
@@ -1 +1 @@
-ALTER TABLE npc_types ADD COLUMN hp_regen_per_second bigint(11) DEFAULT 0 AFTER hp_regen_rate;
+ALTER TABLE npc_types ADD COLUMN hp_regen_per_second bigint(11) DEFAULT 0 AFTER hp_regen_rate;
```

Comment:
To keep from getting other merged PR's showing up in your PR you can do a rebase

```
git pull --rebase origin master
```

Merge conflicts if any eg

```
CONFLICT (content): Merge conflict in zone/zonedb.cpp
error: could not apply f22c1321f... Refactor / simplify NPC loading
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
```

Then when fully rebased, push

```
git push --force origin commands/peqzone_flags
```

Reply:
Gotcha.